src/main/java/frc/robot/commands/arm/ExtendToPosition.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands.arm;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.FourBarArm;
+
+public class ExtendToPosition extends CommandBase {
+  FourBarArm m_arm;
+  PIDController m_pid;
+  double m_armSetpoint;
+  public ExtendToPosition(FourBarArm arm, PIDController pid, double setpoint) {
+    m_arm = arm;
+    m_pid = pid; 
+    m_armSetpoint = setpoint;
+  }
+  @Override
+  public void initialize() {
+    m_pid.reset();
+    m_arm.setEncoderPosition(0);
```

Comment:
Don't reset the encoders here, otherwise it won't work if the arm is in a different angle. I would that run in the init of the subsystem.